github-actions/oss-readme-template/dist/index.js

```diff
@@ -212,40 +216,41 @@ exports.verifySdkSectionHeaders = {
                 content: 'Tuning',
             },
         ];
+        let lastSuccessfulMatch = '';
         for (const expectedHeader of expectedRemainingHeaders) {
-            const nextActualHeader = headers.shift();
+            let nextActualHeader = headers.shift();
             console.info(`Checking for expected header: '${expectedHeader.content}'`);
             console.info(`Next actual header: '${JSON.stringify(nextActualHeader)}'`);
+            while (nextActualHeader !== undefined) {
+                if (expectedHeader.content !== nextActualHeader.content) {
+                    console.info(`Found extra, non-required header: '${nextActualHeader.content}'.  Ignoring.`);
```

Comment:
Could also be a duplicate of a required header

Reply:
That's true.  We can update it if it proves to be confusing.